### PR TITLE
docs(cn): fix api/MultiCompiler translate

### PR DESCRIPTION
**Pull request by Janlaywss**
在源码中，MultiCompiler 实际上是创建了多个 Compiler 实例。这里修正后阅读起来会更顺畅

_describe your changes..._

- [ ] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [ ] Make sure your PR complies with the [writer's guide][2].
- [ ] Review the diff carefully as sometimes this can reveal issues.
- [ ] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

**Comment by Janlaywss** [comment]
@QC-L @dear-lizhihua @yuqingc

File: src/content/api/node.mdx [hunk 1]
```diff
@@ -265,9 +265,9 @@ webpack({
 
 ## MultiCompiler {#multicompiler}
 
-`MultiCompiler` 模块可以让 webpack 在单个 compiler 中执行多个配置。
+`MultiCompiler` 模块可以让 webpack 在单次编译中执行多个配置。
```
Comment (QC-L) [inline src/content/api/node.mdx]:
```suggestion
`MultiCompiler` 模块可以让 webpack 同时执行多个配置。
```

File: src/content/api/node.mdx [hunk 2]
```diff
@@ -265,9 +265,9 @@ webpack({
...
 如果传给 webpack 的 Node.js API 的 `options` 参数，
-是一个由配置对象构成的数组，则 webpack 会应用单独 compiler，
+是一个由配置对象构成的数组，则 webpack 会创建单独的 compiler 实例，
```
Comment (QC-L) [inline src/content/api/node.mdx]:
```suggestion
该参数由是由多个配置对象构成的数组，webpack 会相应地创建多个 compiler 实例，
```

**Comment by QC-L** [comment]
@Janlay884181317 优化了下

**Review by QC-L — APPROVED** [review]

**Pull request merged**